packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Utilities/StringExtensions.cs

```diff
@@ -80,6 +80,11 @@ public static string ToCleanName(this string name, bool isCamelCase = true)
 
             return nameBuilder.ToString();
         }
+
+        [return: NotNullIfNotNull("name")]
```

Comment:
could this be better if
```suggestion
        [return: NotNullIfNotNull(nameof(name))]
```